doc/api/next_api_changes/behavior/23516-MS.rst

```diff
@@ -0,0 +1,3 @@
+Warning when scatter plot color settings discarded
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When making an animation of a scatter plot, if you don't set `c` (the color value parameter) when initializing the artist, the color settings are ignored. `scatter` now raises a warning if color-related settings's are changed without setting `c`
```

Comment:
The issue here is that in rst (unlike markdown) single backticks are cross-references not code formatting.  As `c` is a keyword argument, we have a convention of using bold (`*c*`) to indicated keywords.  The refernce to scatter should be changed to `.Axes.scatter` so that it will cross link to the actual method.